src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -611,7 +611,8 @@
               Grid.Row="1"
               Grid.RowSpan="2"
               Grid.Column="1"
-              Visibility="{x:Bind local:GraphingCalculator.ShouldDisplayPanel(IsSmallState, SwitchModeToggleButton.IsOn, x:False), Mode=OneWay}">
+              Visibility="{x:Bind local:GraphingCalculator.ShouldDisplayPanel(IsSmallState, SwitchModeToggleButton.IsOn, x:False), Mode=OneWay}"
```

Comment:
Same comment here

Reply:
I ended up removing this entirely because it was causing some weird layout issues in small mode